agent/api/task/task.go

```diff
@@ -1737,8 +1737,8 @@ func (task *Task) getSSMSecretsResource() ([]taskresource.TaskResource, bool) {
 	return res, ok
 }
 
-// PopulateSecretsAsEnv appends the container's env var map with secrets
-func (task *Task) PopulateSecretsAsEnv(container *apicontainer.Container) *apierrors.DockerClientConfigError {
+// PopulateSecretsAsEnvAndLogDriver appends the container's env var map with secrets
+func (task *Task) PopulateSecretsAsEnvAndLogDriver(hostConfig *dockercontainer.HostConfig, container *apicontainer.Container) *apierrors.DockerClientConfigError {
```

Comment:
Can we just call it `PopulateSecrets` so we that don't have to change the name again when new secret type is added?